content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,173 @@
+.. image:: discuss/conversation-actions.png
+   :alt: View of the conversation actions.
 
-.. image:: discuss/channels-direct-messages.png
-   :height: 400
-   :alt: View of Discuss's panel emphasizing the titles channels and direct messages in Odoo
-         Discuss.
+Click :icon:`fa-bell` :guilabel:`Notification Settings` to set up notification preferences for the
+conversation, or click :icon:`fa-phone` :guilabel:`Start a Call` to begin a meeting. See the
+:ref:`Meetings <discuss/meetings>` section for more information about meetings.
 
-A company can also easily create :doc:`public and private channels <discuss/team_communication>`.
+Click :icon:`fa-search` :guilabel:`Search Messages` to search messages in the conversation.
 
-Mentions in the chat and on the chatter
----------------------------------------
+Clicking :icon:`fa-thumb-tack` :guilabel:`Pinned Messages` shows pinned messages, and clicking
+:icon:`fa-paperclip` :guilabel:`Show Attachments` shows attachments shared in the conversation.
 
-To mention a user within a chat or the chatter, type `@user-name`; to refer to a channel, type
-`#channel-name`. The user mentioned will be notified in their *inbox* or through an email, depending
-on their communication settings.
+Click :icon:`fa-user-plus` :guilabel:`Add Users` to invite more people to the direct message
+conversation. Click :icon:`fa-cog` :guilabel:`Show Call Settings` to configure voice chat settings.
 
-.. image:: discuss/chat-windows.png
-   :alt: View of a couple of chat window messages for Odoo Discuss.
+At the top of the direct message window, click the name of the direct message to change the group
+name, and choose to add a description in the adjacent :guilabel:`Add a description` field.
 
-.. tip::
-   When a user is mentioned, the search list (list of names) suggests values first based on the
-   task's followers, and secondly on employees. If the record being searched does not match with
-   either a follower or employee, the scope of the search becomes all partners.
+.. note::
+   The :guilabel:`Add a description` field is **only** available for group messages with more than
+   two participants.
 
 User status
 -----------
 
-It is helpful to see what colleagues are up to and how quickly they can respond to messages by
-checking their *status*. The status is shown on the left side of a contact's name on the
-:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
+It is helpful to see what colleagues are up to, and how quickly they can respond to messages, by
+checking their status. The status is displayed as a circle in the bottom-right corner of a contact's
+photo.
+
+The color of the circle represents the user's status:
 
 - Green = online
 - Orange = away
 - White = offline
 - Airplane = out of the office
 
-.. image:: discuss/status.png
-   :height: 300
-   :alt: View of the contacts' status for Odoo Discuss.
+Unpin a direct message conversation
```

Comment:
To avoid confusion with "Pinned messages", I think it would be best to stick with the terms Odoo has in the UI
```suggestion
Leave a direct message conversation
```